source/extensions/filters/http/ext_proc/processor_state.cc

```diff
@@ -126,13 +126,18 @@ absl::Status ProcessorState::handleHeadersResponse(const HeadersResponse& respon
       } else if (complete_body_available_ && body_mode_ != ProcessingMode::NONE) {
         // If we get here, then all the body data came in before the header message
         // was complete, and the server wants the body. It doesn't matter whether the
-        // processing mode is buffered, streamed, or partially streamed -- if we get
-        // here then the whole body is in the buffer and we can proceed as if the
-        // "buffered" processing mode was set.
-        ENVOY_LOG(debug, "Sending buffered request body message");
-        filter_.sendBufferedData(*this, ProcessorState::CallbackState::BufferedBodyCallback, true);
-        clearWatermark();
-        return absl::OkStatus();
+        // processing mode is buffered, streamed, or partially buffered.
+        if (bufferedData()) {
```

Comment:
does it equal to "bufferedData() !=nullptr) ?

Reply:
yeah, it is same thing.  bufferedData() returns Instance* pointer.